components/navigation/MenuBlocks.tsx

```diff
@@ -0,0 +1,55 @@
+import { useRouter } from 'next/router';
+
+import type { MenuBlocksProps } from '@/types/navigation/MenuBlocks';
+import { ParagraphTypeStyle } from '@/types/typography/Paragraph';
+
+import LinkComponent from '../link';
+import Paragraph from '../typography/Paragraph';
+import Label from './Label';
+
+/**
+ * @description Component representing a set of menu blocks.
+ * @param {MenuBlocksProps} props - The props for the MenuBlocks component.
+ */
+export default function MenuBlocks({ items = [] }: MenuBlocksProps) {
+  const router = useRouter();
+
+  return (
+    <>
+      {items.map((item, index) => {
+        const isExternalHref = item.href && item.href.startsWith('http');
+
+        return (
+          <LinkComponent href={item.comingSoon ? '' : item.href} key={index}>
+            <a
```

Comment:
Don't use `a` tag under Link component.

Reply:
It is used in the original file, that's why I used it here as well.

Reply:
@vishvamsinh28 
yes, but we were using Next.JS v12 previously. In Next.JS v13+, this is deprecated.
https://nextjs.org/docs/messages/invalid-new-link-with-extra-anchor